package.json

```diff
@@ -29,26 +29,34 @@
   },
   "dependencies": {
     "axios": "^0.15.3",
-    "babel-preset-stage-1": "^6.1.18",
     "babel-core": "^6.2.1",
     "babel-loader": "^6.2.0",
     "babel-preset-es2015": "^6.1.18",
     "babel-preset-react": "^6.1.18",
-    "jsdom": "^8.1.0",
+    "babel-preset-stage-1": "^6.1.18",
+    "body-parser": "^1.17.1",
     "cors": "^2.8.1",
     "express": "^4.15.2",
+    "jsdom": "^8.1.0",
     "lodash": "^3.10.1",
+    "material-ui": "^0.18.1",
+    "muicss": "^0.9.16",
     "nuka-carousel": "^2.0.4",
     "react": "^0.14.3",
+    "react-async-script": "^0.9.1",
     "react-dom": "^0.14.3",
+    "react-google-recaptcha": "^0.9.3",
     "react-page-slider": "^1.2.1",
     "react-redux": "^4.0.0",
     "react-router": "^2.0.1",
     "react-scrollchor": "^2.2.0",
     "react-slick": "^0.14.7",
     "react-swipeable-views": "^0.11.1",
+    "react-textarea-autosize": "^4.3.2",
     "redux": "^3.0.4",
+    "redux-form": "^3.1.7",
     "redux-thunk": "^2.2.0",
+    "sendgrid": "^2.0.0",
```

Comment:
I didn't know what the ^ standed for. I have the version that we used in your code and it works well.